capsules/detector_text_openvino/capsule.py

```diff
@@ -0,0 +1,45 @@
+from vcap import (
+    BaseCapsule,
+    NodeDescription,
+    DeviceMapper,
+    common_detector_options,
+)
+
+from .backend import Backend, OpenVINOModel
+
+
+class Capsule(BaseCapsule):
+    name = "detector_text_openvino"
+    description = "✨ OpenVINO text detector."
```

Comment:
I suggest adding "text detector and reader"

FYI this shows up on the BrainFrame downloads page, so the descriptions are helpful in that respect